crawler-cli/src/main/java/org/gbif/crawler/camtrapdp/converter/CamtrapDpToDwcaCallback.java

```diff
@@ -221,6 +245,41 @@ private static Optional<License> readLicense(Path unpackedDpPath) {
     return Optional.empty();
   }
 
+  @SneakyThrows
+  private static List<Contact> readContributors(Path unpackedDpPath) {
+    List<Contact> contacts = new ArrayList<>();
+
+    File dataPackage = Paths.get(unpackedDpPath.toString(), "datapackage.json").toFile();
+    JsonNode root = MAPPER.readTree(dataPackage);
+
+    for (JsonNode contributorNode : root.path("contributors")) {
+      Contact contact = new Contact();
+
+      setContactFieldFromJsonNode(contributorNode, "firstName", node -> contact.setFirstName(node.asText()));
```

Comment:
In the dataset that we have in UAT I see contributors like:

```
{
    "title" : "Peter Desmet",
    "path" : "https://orcid.org/0000-0002-8442-8025",
    "email" : "peter.desmet@inbo.be",
    "organization" : "Research Institute for Nature and Forest (INBO)",
    "role" : "contributor"
  }
  ```

and it doesn't match this mapping, are we sure about this mapping?

Reply:
Need to check why they're not set. Those fields are properly set in the `datapackage.json` inside the archive

Reply:
Those are newly added fields so were not present for some datasets maybe. I've published a new version and they're all set

Reply:
Example https://ipt3.gbif-uat.org/resource?r=mica-full